pkg/mcs/resourcemanager/server/token_buckets.go

```diff
@@ -139,6 +147,13 @@ func (gts *GroupTokenBucketState) balanceSlotTokens(
 		}
 	}
 
+	for clientUniqueID, slot := range gts.tokenSlots {
```

Comment:
Do we need check this every handling request?

Reply:
should refine so we can check every 10mins

Reply:
Add `gts.lastCheckExpireSlot` as a guard